volumes/miovision/api/readme.md

```diff
@@ -237,9 +241,15 @@ Since there is this function on Airflow script to send slack notifications when
 
 The Airflow uses BashOperator and run one task named `pull_miovision` using a bash command that looks something like this bash_command = `'/etc/airflow/.../intersection_tmc.py run-api --path /etc/airflow/.../config.cfg --dupes'`. `--dupes` is used to catch duplicates and fail the script when that happen.
```

Comment:
i know its just an example but the bash command's has been updated with a new path file, if you can update it here as well it would be greeeat

Reply:
can we change the `etc` to `data`?